src/pages/VariantPage.js

```diff
@@ -96,6 +174,32 @@ const VariantPage = ({ match }) => (
         ) : null;
       }}
     </Query>
+    <hr />
+    <Heading>Associated index variants</Heading>
+    <SubHeading>
+      Which index variants and studies are linked to this tag variant?
+    </SubHeading>
+    <Query query={associatedIndexesQuery}>
+      {({ loading, error, data }) => {
+        console.log('data', data);
+        return hasAssociatedIndexVariants(data) ? (
+          <AssociatedIndexVariantsTable data={data} />
+        ) : null;
+      }}
+    </Query>
+    <hr />
+    <Heading>Associated tag variants</Heading>
+    <SubHeading>
+      Which tag variants and studies are linked to this index variant?
+    </SubHeading>
+    <Query query={associatedTagsQuery}>
+      {({ loading, error, data }) => {
+        console.log('data', data);
+        return hasAssociatedTagVariants(data) ? (
+          <AssociatedTagVariantsTable data={data} />
```

Comment:
Here as well we could directly pass `data.tagVariantsAndStudiesForIndexVariant.rows` directly to the component. What do you think?

Reply:
:thumbsup: